packages/design-tokens/package.json

```diff
@@ -33,6 +34,7 @@
     "@types/lodash": "^4.14.161",
     "babel-loader": "^8.1.0",
     "clean-webpack-plugin": "^3.0.0",
+    "jest": "^26.6.3",
```

Comment:
Do you think it might be worth syncing all `jest` dependencies across the packages?